addon/webextension/selector/ui.js

```diff
@@ -322,11 +322,98 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
 
   iframePreSelection.onResize = watchFunction(onResize.bind(iframePreSelection));
 
+  let iframePreview = exports.iframePreview = {
+    element: null,
+    document: null,
+    sizeTracking: {
+      windowDelayer: null
+    },
+    display(installHandlerOnDocument, standardOverlayCallbacks) {
+      return new Promise((resolve, reject) => {
+        if (!this.element) {
```

Comment:
It would be really nice to put this stuff in a function – we really should have done this when we had 2 iframes, and then 3, and now 4...

OTOH, it might be better to do that separately with a commit that doesn't change any functionality.